lib/src/trailer.rs

```diff
@@ -107,6 +109,11 @@ fn parse_trailers_impl(body: &str) -> (Vec<Trailer>, bool, Option<String>) {
             let value = multiline_value.iter().rev().join("\n");
             multiline_value.clear();
             trailers.push(Trailer { key, value });
+        } else if let Some(groups) = cherry_re.captures(line) {
+            let key = groups[1].to_string();
+            let value = groups[2].to_string();
+            multiline_value.clear();
+            trailers.push(Trailer { key, value });
```

Comment:
If I understand correctly, a "cherry picked" line shouldn't be parsed as trailer. It should just contribute to the detection of the trailer paragraph?

```sh
% git interpret-trailers --only-trailers --only-input <<'EOF'

foo: bar
(cherry picked from commit 123)
EOF
foo: bar
```

To implement this, it's probably better to first scan description text backwards to find the start line, then parse trailer lines forward from there. The stricter version `parse_trailers()` doesn't need the first step, and would just reject any bad lines.

Reply:
I thought it could be helpful to have the cherry-pick trailer parsed as a trailer.
Not for this PR though, I'll remove it